CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Development / Bugs / Support
+
+If you want to help contribute to this plugin or find a bug, please [file an issue] on Github, and/or submit me a PR.  This is Open Source, so don't expect me to fix bugs immediately, but I'll try my best to reasonably support this plugin.  Contributors are always welcome though.
+
+To get a development environment up will need a recent golang installed and setup.  Then with a single make command below command it will build, install, and try to run the example at [examples/orka.pkr.hcl](./examples/orka.pkr.hcl).  You may need to edit this file to have the source VM config that you have locally, as it is hardcoded to my environment's image `macos-catalina-10-15-5` at the moment.
+
+```bash
```

Comment:
In case the above suggestion is accepted, the commands below can be deleted.

Reply:
Done